src/citrine/resources/project.py

```diff
@@ -599,9 +602,57 @@ def register(self, name: str, *, description: Optional[str] = None) -> Project:
         project = Project(name, description=description)
         return super().register(project)
 
+    def _list_base(self, *, per_page: int = 1000, archived: Optional[bool] = None):
+        filters = {}
+        if archived is not None:
+            filters["archived"] = str(archived).lower()
+
+        fetcher = partial(self._fetch_page, additional_params=filters, version=self._api_version)
+        return self._paginator.paginate(page_fetcher=fetcher,
+                                        collection_builder=self._build_collection_elements,
+                                        per_page=per_page)
+
     def list(self, *, per_page: int = 1000) -> Iterator[Project]:
```

Comment:
Note that `ProjectCollection.list` defaults to listing every project. This differs from many other collections, such as predictors an design spaces, where `list` only lists the active assets. This will be addressed in 4.0 by renaming `PredictorCollection.list` to `list_active` and `PredictorCollection.list_all` to `list`.